docs/Scheduler.md

```diff
@@ -0,0 +1,350 @@
+## What is
+Objectively, a **Scheduler** is 1:1 to **Kubernetes namespace** (as referenced [here](Kubernetes.md)), and contains all the information for creating 
```

Comment:
Hey @caiordjesus I think that as Kubernetes is a port only related to our runtime we should avoid relating its concepts with Maestro concepts. What do you think about describing what a Scheduler is for Maestro Next only?